modules/gateways/monero/createinvoice.php

```diff
@@ -42,9 +44,10 @@ function monero_payment_id(){
 
 $secretKey = $GATEWAY['secretkey'];
 $hash = md5($invoice_id . $payment_id . $amount_xmr . $secretKey);
-echo "<link href='/modules/gateways/monero/style.css' rel='stylesheet'>";
-echo  "<script src='https://code.jquery.com/jquery-3.2.1.min.js'></script>";
-echo  "<script src='/modules/gateways/monero/spin.js'></script>";
+echo "<link href='$system_url/modules/gateways/monero/style.css' rel='stylesheet'>";
+echo  "<script src='https://code.jquery.com/jquery-3.6.0.min.js'></script>";
+echo  "<script src='$system_url/modules/gateways/monero/spin.js'></script>";
+echo  '<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.10/clipboard.min.js"></script>';
```

Comment:
Better to have `echo "<script src='..'>"`, just to keep consistency. We might want to open a new issue for creating code style among all the monero-integrations repositories.

Reply:
> Some minor changes. What I was wondering about was the image for clipped action. Where did you take it from? Can we use it without breaking any license?

It looks like it is from this Image : https://clipboardjs.com/assets/images/clippy.svg
From their site: https://clipboardjs.com/
That was converted to png. I'm not sure on the license

I'm pretty sure though we could use whatever image from free online site in a worst case scenario

Reply:
> Better to have `echo "<script src='..'>"`, just to keep consistency. We might want to open a new issue for creating code style among all the monero-integrations repositories.

Done as requested. :)